docs/asm-gke-terraform/main.tf

```diff
@@ -29,24 +28,22 @@ resource "google_container_cluster" "cluster" {
     google_project_service.project
   ]
 }
-data "google_project" "project" {
-  project_id = var.project_id
-}
+
+data "google_project" "project" {}
```

Comment:
**Question:** Do we still need this?
```suggestion
```
Feel free to ignore. This is a nit.

Reply:
It's needed to add the project number to the mesh_id label on the cluster. I hope that one day soon the label won't be needed anymore.